switchlink/switchlink_neigh.c

```diff
@@ -123,11 +123,6 @@ void switchlink_process_neigh_msg(const struct nlmsghdr* nlmsg, int msgtype) {
   } else {
     intf_h = ifinfo.intf_h;
   }
-  switchlink_handle_t bridge_h = g_default_bridge_h;
-  if (ifinfo.intf_type == SWITCHLINK_INTF_TYPE_L2_ACCESS) {
```

Comment:
why are we removing this block?

Reply:
clang-tidy flagged `bridge_h` as an unreferenced variable. It's only used in the `krnlmon_assert` statement, which is disabled unless we build krnlmon in Debug mode. I didn't see this as a compelling reason to preserve what is otherwise dead code.